### PR TITLE
Bakery conf for spark-history-server

### PR DESCRIPTION
**Pull request by darkjh**

### CODE REVIEW

File: _bakery/spark-history-server/files/launch-spark-history-server.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+export SPARK_HISTORY_OPTS="-Dspark.history.fs.logDirectory=s3a://teads-hadoop/engine/spark-logs -Dspark.history.fs.cleaner.enabled=true"
```
Comment (darkjh) [inline _bakery/spark-history-server/files/launch-spark-history-server.sh]: We need to setup in terraform so that the machine has acces to s3, right?

Comment (pdalpra) [reply _bakery/spark-history-server/files/launch-spark-history-server.sh]: yes

Comment (darkjh) [reply _bakery/spark-history-server/files/launch-spark-history-server.sh]: Do I need to create a jenkins job for it in the jenkins-dsl?

**Comment by darkjh** [comment]
retest

**Comment by darkjh** [comment]
retest

**Comment by darkjh** [comment]
retest

**Pull request merged**